keyboards/m65/rev1/halconf.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2021 QMK
```

Comment:
I believe this file can be deleted, as it doesn't contain any meaningful changes vs. the default halconf for this MCU.

Reply:
you are write on this removed it